### ISSUE TITLE
about the validation set in the training process

### ISSUE THREAD

**Issue by ynw2021** [autonomousvision/tuplan_garage#22]
Hi,

I am a little confused about the val14 data split. Did you use all the validation set and the training set for sampling the 178k samples for the train150 set, in which the samples in the val set is used to do validation, then the val14 set with 1118 samples from the val set is used to test the performance? That means the val set and the test set of val14 setting might overlap？

**Comment by DanielDauner** [comment]
Thanks for bringing this up. The training pipeline from nuPlan utilizes training and validation samples from the corresponding splits. The validation samples aren’t used for training and only serve to calculate metrics and identify overfitting in the training loop. Our benchmark comprises approximately 160k training and 15k validation samples.

Actually, the scenarios from Val14 and the internal validation split of the training loop can temporally overlap. I checked the scenario tokens of both validation subsets with a script and found some overlapping samples. It is only fractional since Val14 exclusively queries challenge types and ensures that scenarios are 15s apart. If you want to be absolutely safe, consider using the test set for performance evaluation or simply ignore the validation split in the training loop.

I hope this clears up any confusion.

**Comment by ynw2021** [comment]
Thanks for your clarification，my confusions are cleared!
